src/release/issue-triaging.md

```diff
@@ -134,6 +135,10 @@ There are many different labels that can be applied to issues.
     - `S-has-bisection`: A bisection has been performed and the bisection result is convincing.
     - `S-waiting-on-LLVM`: Waiting on upstream LLVM PR or fix.
     - `S-tracking-forever`: The tracking issue is never intended to be closed.
+- [`beta-nominated`]: Tracks changes which need to be [backported to beta].
+- [`stable-nominated`]: Tracks changes which need to be [backported to stable] (in anticipation of a point release).
```

Comment:
Nit: it's more like nominated to be discussed by relevant team(s) as a {beta,stable}-backport candidate. If the relevant team(s) approve the {beta,stable}-backport, then T-release will handle the backport (usually).